### ISSUE TITLE
judge-data-entry-add-apprenticed

### ISSUE THREAD

**Issue by purselane** [freelawproject/courtlistener#1907]
Currently, there's no option in the Education section to indicate that someone became a judge by apprenticing rather than by going to law school.

_Originally posted by @purselane in https://github.com/freelawproject/courtlistener/issues/1905#issuecomment-1039764777_

**Comment by mlissner** [comment]
Thanks @purselane. Do you have an example of this? I guess the fix here is going to be hard, b/c the school field is required on the Education table, but if you had an example, it'd help when we eventually got to this.

**Comment by purselane** [comment]
Yes, Marcos R Camacho (P 178) did not attend law school and earned his law degree by apprenticing and participating in the Read Law/Law Office Study Program, which is run by the State Bar of California.
